### PR TITLE
Move in-tree extensions to top-level folder

### PR DESCRIPTION
**Pull request by jakolehm**
We should keep them logically separated from app sources.

### CODE REVIEW

**Review by nevalla — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "compile:main": "webpack --config webpack.main.ts",
     "compile:renderer": "webpack --config webpack.renderer.ts",
     "compile:i18n": "lingui compile",
+    "compile:extensions": "rollup --config src/extensions/rollup.config.js",
```
Comment (ixrock) [inline package.json:23]: Let's re-use in line 18 similar to `dev:main`, etc? 
```
"dev:extensions": "yarn compile:extensions --watch"
```

**Review by ixrock — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (jakolehm) [reply package.json:23]: Yes, fixed.

**Pull request merged**